### PR TITLE
Ka conflict marge

### PR DESCRIPTION
**Pull request by Marouann**

### CODE REVIEW

**Comment by pierlj** [comment]
Conflicts that happens when there is a cell between two agents:
```
=====
 0 1
=====
```
Example of such conflicts in MAExampleImparDist.lvl
are not solved yet. It might be due to to the way we get un_met_preconditions.

File: client/action.py [hunk 1]
```diff
@@ -75,5 +75,5 @@ def execute(self, s: 'State', variables):
     'NoOp',
     lambda agt, agtFrom: [Atom('AgentAt', agt, agtFrom)],
     lambda agt, agtFrom: [Atom('AgentAt', agt, agtFrom)],
-    lambda agt, agtFrom: [Atom('AgentAt', agt, agtFrom)],
+    lambda agt, agtFrom: [Atom('AgentAt', agt, agtFrom), Atom('Free', agtFrom)],
```
Comment (carlomarxdk) [inline client/action.py:78]: Why does it free the cell if the agent is still there?

**Review by carlomarxdk — DISMISSED** [review]
There are some inconsistencies with the NoOp

**Review by carlomarxdk — APPROVED** [review]

File: client/action.py (hunk 1 above)
Comment (kamranthomas) [reply client/action.py:78]: It is one of the negative effects, so its ¬Free(agtFrom) which makes sense. But maybe AgentAt should be removed as a negative effect

Comment (kamranthomas) [reply client/action.py:78]: But right now NoOp is not being applied as an action, so it shouldn't effect the rest of the code yet...

**Pull request merged**